Frontend/src/components/Guess.jsx

```diff
@@ -0,0 +1,300 @@
+import React, { useContext, useState } from 'react'
+import Letter from './Letter'
+
+
+
+export default function Guess(props) {
+  const { length, currentGuess } = props
```

Comment:
Isn't it possible to derive the `length` from the `guess`? If so, the additional property (`length`) is derived and should be removed. Otherwise, you might end up in states where someone passes in a five-letter word but sets `length` to 10.